aiounifi/models/configuration.py

```diff
@@ -18,6 +18,7 @@ class Configuration:
     password: str
     port: int = 8443
     site: str = "default"
+    for_hotspot: bool | None = None
     ssl_context: SSLContext | Literal[False] = False
```

Comment:
```suggestion
    ssl_context: SSLContext | Literal[False] = False
    for_hotspot: bool = False
    # Comment here about what this is
```
No need for None in this case
Please add a description to this as to what it does